gateway/src/main/java/discord4j/gateway/IdentifyOptions.java

```diff
@@ -33,6 +34,8 @@
     @Nullable
     private final StatusUpdate initialStatus;
 
+    private final long intents;
```

Comment:
This feature should remain optional if we target v6. That means not changing anything about how the identify payload is currently sent.

Make this field nullable Long